src/main/java/kg/jarkyn/cobspecserver/Response.java

```diff
@@ -0,0 +1,13 @@
+package kg.jarkyn.cobspecserver;
+
+public class Response {
+    private String status;
+
+    public Response(String status) {
+        this.status = status;
+    }
+
+    public byte[] getBytes() {
```

Comment:
(already mentioned in person, adding here for completeness)
i see what you mean, but `Response will not be created with strings only, the body will be generated as bytes, and I want it to be responsibility of Response to present response as one or the other :)